contrib/seeds/makeseeds.py

```diff
@@ -160,10 +164,14 @@ def main():
     mns = filtermultipayoutaddress(mns)
     # Extract IPs
     ips = [parseip(mn['state']['service']) for mn in mns]
+    for onion in onions:
+        parsed = parseip(onion)
+        if parsed is not None:
+            ips.append(parsed)
     # Look up ASNs and limit results, both per ASN and globally.
     ips = filterbyasn(ips, MAX_SEEDS_PER_ASN, NSEEDS)
     # Sort the results by IP address (for deterministic output).
-    ips.sort(key=lambda x: (x['net'], x['sortkey']))
+    ips.sort(key=lambda x: (x['net'], x['sortkey']), reverse=True)
```

Comment:
why?

Reply:
So that onion seeds were at the top; don't know if it makes any difference 🤷